region/maridia/inner-green/East Sand Hall.json

```diff
@@ -267,29 +266,40 @@
     },
     {
       "link": [1, 4],
-      "name": "Evir Avoid Suitless",
+      "name": "Suitless HiJump Evir Avoid",
       "notable": false,
+      "entranceCondition": {
+        "comeInRunning": {
+          "speedBooster": "any",
+          "minTiles": 3.4375
+        }
+      },
       "requires": [
         "canSuitlessMaridia",
         "HiJump",
-        {"or": [
-          {"and": [
-            {"adjacentRunway": {
-              "fromNode": 1,
-              "usedTiles": 2.5,
-              "physics": ["normal"]
-            }},
-            "canCarefulJump"
-          ]},
-          {"enemyDamage": {
-            "enemy": "Evir",
-            "type": "particle",
-            "hits": 1
-          }}
-        ]}
+        {"adjacentRunway": {
+          "fromNode": 1,
+          "usedTiles": 2.5,
+          "physics": ["normal"]
+        }},
+        "canCarefulJump"
       ],
       "note": "By entering the room with run speed, it is possible to jump over the pair of Evirs."
     },
+    {
+      "link": [1, 4],
+      "name": "Suitless HiJump, Evir Projectile Hit",
+      "notable": false,
+      "requires": [
+        "canSuitlessMaridia",
+        "HiJump",
```

Comment:
```suggestion
        "HiJump",
        "canPlayInSand",
```

Reply:
Should this be added on the strat above as well, that just jumps over the Evir?